config/webpack.config.dev.js

```diff
@@ -122,6 +122,7 @@ module.exports = {
             loader: require.resolve('babel-loader'),
             options: {
               cacheDirectory: true,
+              presets: ['react-app']
```

Comment:
This is what fixes the babel error, As you can see it is exactly the same as babel config in package.json. But, we have to put it here as well, as babel-loader doesn't recognize config in package.json unfortunately.

Reply:
Nice one, thanks @lebaz20